### PR TITLE
Fix the returned error details,and add the output information of the …

### PR DESCRIPTION
**Pull request by liutao841204**
Fix the returned error details,This error is located in https://github.com/alibaba/sealer/blob/main/build/context.go#L54

### CODE REVIEW

File: build/context.go [hunk 1]
```diff
@@ -50,9 +50,11 @@ func (c *CloudBuilder) sendBuildContext() (err error) {
 	}()
 	// send to remote server
 	workdir := fmt.Sprintf(common.DefaultWorkDir, c.local.Cluster.Name)
-	if err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName); err != nil {
+	err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName)
+	if err != nil {
 		return err
```
Comment (fanux) [inline build/context.go:55]: We want return error details, for example:

return fmt.Errorf("failed to copy tar file: %s, err: %v", tarFileName, err)

This format is fine
```
 if err =xxx; err != nil {
...
}
```

File: build/context.go [hunk 2]
```diff
@@ -50,9 +50,11 @@ func (c *CloudBuilder) sendBuildContext() (err error) {
...
 	}
+	logger.Info("the program is sending the %s file!", tarFileName)
```
Comment (fanux) [inline build/context.go:57]: We needn't add log here, because the Copy funcion already has log.

Comment (liutao841204) [reply build/context.go:57]: OK

File: build/context.go [hunk 3]
```diff
@@ -50,7 +50,8 @@ func (c *CloudBuilder) sendBuildContext() (err error) {
 	}()
 	// send to remote server
 	workdir := fmt.Sprintf(common.DefaultWorkDir, c.local.Cluster.Name)
-	if err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName); err != nil {
+	err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName)
```
Comment (fanux) [inline build/context.go:53]:
```suggestion
	if err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName); err != nil {
	    return fmt.Errorf("failed to copy tar file: %s, err: %v", tarFileName, err)
	}
```

**Pull request closed without merging**